File: TugasPraktikum04/Bahasa Python/PRAK405-2310817320013-AikoAnatashaWendiono.py
a, b = map(int, input("Input: ").split())
total = 0
hasil = 0

for i in range(1, a + 1):

    for j in range(i, 0, -1):
        print(f"({j} * {b})", end="")
        
        nilai = j * b
        hasil += nilai
        total += nilai

        if (j > 1):
            print(" + ", end="")

    print(f" = {hasil}")
print(total)

File: TugasPraktikum04/Bahasa C/PRAK405-2310817320013-AikoAnatashaWendiono.c
#include <stdio.h>

int main(){
    int a, b, j, i, total, hasil, nilai;

    printf("Input: ");
    scanf("%d %d", &a, &b);

    for (i = 1; i <= a; i++){
        hasil = 0;
        total = 0;

        for (j = i; j >= 1; j--) {
            printf("(%d * %d)", j, b);

            nilai = j * b;
            (hasil += nilai); (total += nilai);
            if (j > 1){
                printf(" + ");
            }
        }
        printf(" = %d\n", hasil);
    }
    printf("%d\n", total);
    return 0;
}
